CONTRIBUTING.adoc

```diff
@@ -155,6 +155,45 @@ targets:
       - addPrefix: "olblak/polls@256:"
 ```
 
+=== TEST
+
+==== Unit Tests
+
+TBD.
+
+[source,bash]
+----
+make test-short
+----
+
+==== Integration Tests
+
+[source,bash]
+----
+make test
+----
+
+==== End to End (e2e) Tests
+
+* https://github.com/ovh/venom[OVH's Venom CLI] is required
+* A https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token[GitHub Personnal Access Token (PAT)] is required with read access to public repositories
+* Define the following environment variables (otherwise the `make` command will fail with a message telling you which variable is missing):`
+** `$GITHUB_TOKEN` set to the GitHub PAT mentioned above
+** `$GITHUB_ACTOR` set to the GitHub username associated to the aforementioned GitHub PAT
+
+
+[source,bash]
+----
+export GITHUB_TOKEN=$(<command to get token from password manager>) # Treat this as a SENSITIVE value!
+export GITHUB_ACTOR=xxxxx
+
+venom version # expect v1.1.0+
```

Comment:
Should an updatecli manifest be added to keep this version up to date? ^^

Reply:
There is already one that could be extended https://github.com/updatecli/updatecli/blob/main/updatecli/updatecli.d/venom.yaml